docs/ibc/middleware/develop.md

```diff
@@ -2,7 +2,7 @@
 order: 1
 -->
 
-# IBC Middleware
+# IBC middleware
```

Comment:
Personally I prefer the capitalization on these words so it is `IBC Middleware` and also in the subsequent headings but up to you

Reply:
I try to follow [Google's developer documentation style guide](https://developers.google.com/style/headings) recommendation about headings: `Use sentence case for headings and titles.`

I also personally prefer this style, but I don't want also to enforce it if everybody else doesn't agree. I just think it would be good to have a common style, so that we are consistent all across our docs.

Reply:
Fair enough. I agree consistency matters over any particular decision. Im ok to follow the style guide you linked

Reply:
@crodriguezvega maybe we can add the style guide you linked somewhere for future reference? Maybe in the `DOCS_REAMDE.md`